conans/test/integration/toolchains/cmake/test_ninja.py

```diff
@@ -123,15 +149,15 @@ def test_locally_build_windows_debug(self):
             os=Windows
             arch=x86
             compiler=Visual Studio
-            compiler.version=15
+            compiler.version=16
```

Comment:
No need to change it, this is already working and passing CI correctly

Reply:
Yes, I only changed to work on my environment. going revert